cypress/support/commands.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @copyright Copyright (c) 2019 John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @author John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+ 
+ const url = Cypress.config('baseUrl').replace(/\/index.php\/?$/g, '')
+ Cypress.env('baseUrl', url)
+ 
+ Cypress.Commands.add('login', (user, password, route = '/apps/deck/') => {
+	let session = `${user}-${Date.now()}` 
```

Comment:
**Question:**
Why did you use `Date.now()`?
It will ways generate a new session instead to reuse the previous session